test/kitchen/tasks/run-test-kitchen.sh

```diff
@@ -82,6 +82,11 @@ if [ -z ${AGENT_VERSION+x} ]; then
   popd
 fi
 
+# Generate a password to use for the windows servers
+if [ -z ${SERVER_PASSWORD+x} ]; then
+  export SERVER_PASSWORD=$(date +%s | sha256sum | base64 | head -c 32 ; echo)
```

Comment:
this password is only determined by the unix epoch (in seconds) at which the command was run. Could we use `/dev/urandom` directly instead? Something like `< /dev/urandom tr -dc A-Za-z0-9 | head -c32` should work

Reply:
Sure, that's better